core/src/main/scala/org/bitcoins/core/wallet/utxo/UTXOSpendingInfo.scala

```diff
@@ -57,6 +58,16 @@ sealed abstract class UTXOSpendingInfo {
   def redeemScriptOpt: Option[ScriptPubKey]
 
   def scriptWitnessOpt: Option[ScriptWitness]
+
+  def conditionalPath: ConditionalPath
```

Comment:
Why does this have to be defined on every `UTXOSpendingInfo`? It seems to be it should just need to be defined `ConditionalSpendingInfo`?

Reply:
Any spend with nesting (P2SH, P2WSH, Locktime, etc.) can contain a `ConditionalScriptPubKey` within it